notebooks/203-meter-reader/203-meter-reader-digital.ipynb

```diff
@@ -0,0 +1,1085 @@
+    "            norm_img_batch = batch_text_box(\n",
+    "                img_crop_list, img_num, indices, beg_img_no, batch_num)\n",
+    "\n",
+    "            # Run inference for text recognition. \n",
+    "            rec_results = rec_compiled_model([norm_img_batch])[rec_output_layer]\n",
+    "\n",
+    "            # Postprocessing recognition results.\n",
+    "            rec_result = text_decoder(rec_results)\n",
+    "            for rno in range(len(rec_result)):\n",
+    "                rec_res[indices[beg_img_no + rno]] = rec_result[rno]   \n",
+    "            if rec_res:\n",
+    "                txts = [rec_res[i][0] for i in range(len(rec_res))] \n",
+    "\n",
+    "        # record the recognition result\n",
+    "        struct_result[key] = txts[0]\n",
+    "        \n",
+    "    return struct_result\n",
+    "\n",
+    "struct_result = main_for_field_strength_meter(img, DESIGN_LAYOUT, RESULT_TEMP, rec_compiled_model, rec_output_layer, text_decoder)\n",
+    "        \n",
+    "# the raw output information\n",
+    "print(struct_result)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "92afe33a-9218-4c1a-941e-931e087265e0",
+   "metadata": {},
+   "source": [
+    "### Postprocessing the output information\n",
+    "For correcting the error in recognition, we only need to configure a `dict`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fc916df8-d486-47f5-8318-4eb2e32fb2e4",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# Congiguration for postprocessing of the results\n",
+    "RESULT_POST = {\"Info_Probe\":[['MP', 'LF', '探头:LF-01']],  # words need to be mapped\n",
+    "               \"Freq_Set\":[['RP', '实时值', ''], ['RP', ' ', '']],  # words need to be replace\n",
+    "               \"Val_Total\":[['RP', 'H2', 'Hz']],\n",
+    "               \"Val_X\":[['RP', 'X', ''], ['RP', ':', '']], \n",
+    "               \"Val_Y\":[['RP', 'Y', ''], ['RP', ':', '']], \n",
+    "               \"Val_Z\":[['RP', 'Z', ''], ['RP', ':', '']], \n",
+    "               \"Unit\":[['MP', 'T', 'μT'],['MP', 'kV', 'kV/m'],['MP', 'kv', 'kV/m'],['MP', 'vm', 'V/m'],['MP', 'Vm', 'V/m'],['MP', 'A', 'A/m']], \n",
+    "               \"Field\":[]}  # nothing need to do"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4fca419b-3fe8-47ef-9151-c32335729499",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Post-processing, fix some error made in recognition\n",
+    "struct_result = post_processing(struct_result, RESULT_POST)\n",
+    "\n",
+    "# Print result\n",
+    "print(struct_result)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "443a7b45-cfc7-400e-897b-5fdff2c78042",
+   "metadata": {},
+   "source": [
+    "## Main Function with Segment code LCD\n",
+    "\n",
+    "Here shows how to extract the value in the screen of domestic intelligent meter.\n",
```

Comment:
Can you ensure name of "domestic intelligent meter" is same through this notebook